src/olympia/scanners/tests/test_models.py

```diff
@@ -238,6 +238,15 @@ def test_get_files_by_matched_rules_for_yara(self):
             rule2: [file2],
         }
 
+    def test_get_files_by_matched_rules_no_file_somehow(self):
```

Comment:
How is this actually possible?

Reply:
It was happening in tests and with manually tweaked data. Since most of the code did handle cases where some data was missing, I thought I might as well add that fallback as well.

Reply:
ok. I guess that's fine. I don't like the `???` notation but hopefully that will never appear :p

Reply:
I'm open to suggestions :) Didn't want to go for a blank string, wanted *something* to be shown